app/pandas.ipynb

```diff
@@ -549,14 +1385,18 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "outputs": [],
    "source": [
```

Comment:
The Pandas solution for calculating the moving average is missing. You can use the `rolling` method with `mean` to achieve this. For example: `data_series.rolling(window=5).mean().dropna()`.